xonsh/ptk_shell/tokenize_ansi.py

```diff
@@ -0,0 +1,28 @@
+from prompt_toolkit import ANSI
```

Comment:
This probably doesn't need to be in its own module since it is only used by shell